### DISCUSSION TITLE
comments in drafts transfered to release

### DISCUSSION THREAD

**Discussion by maxlecalve** [knowledgefutures/pubpub#3031] [Ideas]
Hi pubpub team, 
I'm coming back with the same request of keeping the comments when releasing the pubs. In my practice, it's an important benefit of Pubpub to have these interactions are kept open for the public to see. 

I recently did this classroom experiment, and I was a bit sad to realise that the (few) comments of the students on each other's pubs had been erased when I published. 
https://drawing-people-drawing.pubpub.org/volume-1

A toggle switch in community settings would be neat. Or just enable that function for my communities (all of them: stretchingmaterialities; speculativerealitieslab; curious-ensketchments-moa; drawing-people-drawing)
Many thanks for the amazing amazing work and greetings from Berlin.
Maxime

**Comment by meeloun** [comment]
When choosing homework ghostwriting      [https://www.yydaixie.com](https://www.yydaixie.com)         services, it is necessary to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.

**Comment by amdeis** [comment]
This is very interesting to consider from the point of view of a journal! HDSR uses PubPub and relies on the comments made in drafts (typically by the proofreader and editorial office) being removed upon publication of the article so the public cannot see that step, though we usually archive the draft comments beforehand as a precaution. I can see how this feature is hindering in a classroom setting and thing a toggle community setting switch would be great!

**Comment by maxlecalve** [comment]
Hi Amara,
Indeed, the classroom setting is one thing, but more importantly there is a
branch of social science that relies on interactions with research
subjects, and highlights the value of such open collaborations. My work
with the neurosurgeons is a clear case, but I have a few more collected in
the new "meta" pubpub that I am currently working on, which would give a
home for such collaborative projects in anthropology (including the
classroom).

Anthroreverb.pubpub.org

If not a killer feature, this function would be a great appeal for my
research community. A toggle switch would be great.
Thanks!

Maxime

**Comment by saqi004** [comment]
I’m following up on my request to keep comments visible when releasing pubs. In my recent classroom experiment, I was disappointed to find that the comments from students were erased upon publishing. This feature is crucial for fostering public interactions.

It would be great to have a toggle switch in community settings or simply enable this for my communities (stretchingmaterialities, speculativerealitieslab, [red ribbon menu](https://menuland.ph/red-ribbon-cake-menu-prices-philippines/) curious-ensketchments-moa, drawing-people-drawing).

Thanks again for the great work, and greetings from Berlin!

**Reply by eastofesten** [reply]
Hi Maxime - thanks for following up on this! We don't have a way in PubPub to enable this on a community basis.

Just making sure you've seen our [recent announcement](https://github.com/pubpub/pubpub/discussions/3130) about the shift from PubPub Legacy to PubPub Platform. We don't have plans for this specific feature in the roadmap yet, but it's something we can return to as we migrate communities.

**Comment by maxlecalve** [comment]
Hi Emily,
Thanks for the follow up!
I filled the form and tried to get in touch with you guys regarding the
shift to Platform. I have been using pubpub communities for a long time. I
would like to start a conversation to create a Platform at the destination
of researchers who engage in the same kind of public field work, I am
meeting with Berlin Universities Press this Friday to discuss it.
They don't seem to like your new business model, which is why this could go
into the direction of a pubpub hosting. But I would like to convince them.
It would be great to get in touch.
Can we perhaps have a call next week sometime? Monday or Tuesday would work
for example, before 5pm CET.
Best,
Maxime Le Calvé

___

Matters of Activity. Image Space Material
Cluster of Excellence Humboldt-Universität zu Berlin

Dr. Maxime Le Calvé (he/him)
Research Associate, anthropologist of art and science
Project Leader of “Cutting”
Member of "Object Space Agency"

Co-Founder of the Speculative Realities Lab
<https://speculativerealitieslab.pubpub.org/>(Charité –
Universitätsmedizin Berlin)

NEW: Golden-Pudel-Ethnographie (OA book
<https://www.transcript-verlag.de/978-3-8376-7269-5/golden-pudel-ethnographie/?number=978-3-8376-7269-5>)
/ Field/Works2 (online exhibition <https://fieldworks.easaonline.org/>) /
Cutting Up Writing Up (RAO book
<https://anthroreverb.pubpub.org/cutting-up-writing-up>)

web: maximelecalve.com <https://www.maximelecalve.com/>
web: www.matters-of-activity.de/en/
address: Sophienstrasse 22a, D-10178 Berlin
postal: Unter den Linden 6, D-10099 Berlin
ORCID ID <https://orcid.org/0000-0002-1846-4722>

**Reply by eastofesten** [reply]
Thanks! Someone from PubPub will follow up with you by email.
